src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Always set box-sizing in the `* {...}` selector - Read more: https://css-tricks.com/box-sizing/